.github/ISSUE_TEMPLATE/security_bug_report.md

```diff
@@ -0,0 +1,8 @@
+---
+name: 🔥 Security bug report
```

Comment:
Indeed, not sure if emoticons would help.

Reply:
Emojis won't hurt. Icons improve UX, it's easier to promptly identify the option this way.